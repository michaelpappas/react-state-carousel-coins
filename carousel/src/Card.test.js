import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card.js";

describe('Card', function() {
  test('renders without crashing', function() {
    render(<Card
      caption='test'
      src='test'
      currNum='test'
      totalNum='test'
    />);
  });

  test('matches initial snapshot', function() {
    const { container } = render(<Card
      caption='test'
      src='test'
      currNum='test'
      totalNum='test'
    />);
    expect(container).toMatchSnapshot();
  });
})