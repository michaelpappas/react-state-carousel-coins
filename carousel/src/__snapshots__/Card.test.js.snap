// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card matches initial snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test
    </h4>
    <img
      alt="test"
      class="Card-image"
      src="test"
    />
    <small
      class="Card-small"
    >
      Image 
      test
       of 
      test
      .
    </small>
  </div>
</div>
`;
